plugins/@grouparoo/postgres/src/initializers/plugin.ts

```diff
@@ -31,6 +31,7 @@ import {
   QueryPropertyTemplate,
 } from "@grouparoo/app-templates/dist/source/query";
 
+// eslint-disable-next-line @typescript-eslint/no-var-requires
 const packageJSON = require("./../../package.json");
```

Comment:
We could also use `fs` to load the file if it's better

Reply:
Not sure it is, but an easy change.

Reply:
Breaks things